gosuper/build.sh

```diff
@@ -12,6 +12,9 @@ case "$ARCH" in
 'i386')
 	export GOARCH=386
 ;;
+'iot2000')
+	export GOARCH=386
```

Comment:
we should also set `GO386=387`

Reply:
@petrosagg we already set it in the Dockerfile https://github.com/resin-io/resin-supervisor/blob/master/Dockerfile#L96